### PR TITLE
[13.0][MIG] website_breadcrumb

### PR DESCRIPTION
**Pull request by bjornbillen**

### CODE REVIEW

File: website_breadcrumb/views/layout.xml
Comment (Tardo) [inline website_breadcrumb/views/layout.xml]:
```suggestion
                        <div class="col-12">
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks Good. Code review

File: website_breadcrumb/readme/CONFIGURE.rst
Comment (wytzejan) [inline website_breadcrumb/readme/CONFIGURE.rst]:
```suggestion
#. Go to Website Admin > Configuration > Menus.
```

**Review by wytzejan — COMMENTED** [review]
Nice. I did a functional review and found out that the readme was outdated.

**Review by claudiagn — COMMENTED** [review]
waiting for the runbot to work so I can review it functionally 👍

**Comment by acorpuz** [comment]
@bjornbillen Your changes look interesting, will this PR be merged?

**Review by Tardo — APPROVED** [review]
Looks good. Code review. Thanks!

**Comment by pedrobaeza** [comment]
I have rebase, squash administrative commits and apply the pre-commit commit before the real changes, so merging:

/ocabot merge nobump
/ocabot migration website_breadcrumb

**Pull request merged**
